### DISCUSSION TITLE
Does Neurodesktop support sound?

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#119] [Q&A] [answered]
Just wonder if Neurodesktop can support sound? It doesn't work when I use RDP via the web browser, and I wonder if sound can be enabled, or if it's too much effort?
We have some potential users in Swinburne who work on sonification of EEG signals, so I guess it will be a must for them.

**Comment by stebo85** [comment, marked as answer]
This could be possible and I can’t see why this wouldn’t work. Give it a go and submit a pull request :)
